### ISSUE TITLE
Inline Editable Success() does not work with values of blank/null

### ISSUE THREAD

**Issue by timothymarois** [olifolkerd/tabulator#2335]
**Describe the bug**
When using the inline editor http://tabulator.info/examples/4.4#editable

The `success(input.value);` will not process if the value being sent is blank/null. Which means I'm unable to clear input fields to a blank value. It won't send the info to the server. 

UPDATE: 

It works when the current value is anything except a zero "0". Which means it will not process a null/blank value if the input starts off as a "0"

**Tabulator Info**
4.4

**Comment by timothymarois** [comment]
Also to add, the success() method changes what the input value is, which means I can't even force it to update manually by sending in a temporary value to the server to act as a delete. This needs to be fixed.

**Comment by timothymarois** [comment]
Looks like the input validation is working (because the success method is validating the value and updating the input to a blank) but it's not firing off to `cellEdited()` which means I have no access to send it to the server.

**Comment by timothymarois** [comment]
I don't like this, but due to the current limitations, I have added a "rig" to get around this issue at the moment. 

Currently, checking if it's blank or null and forcing the value to be `XREMOVE`, this way `success()` will actually fire off to `cellEdited()`

```
       function onChange() {
                changeValue = input.value;
                if (input.value===null || input.value==='') changeValue = 'XREMOVE';

                if(input.value !== inputValue) {
                    success(changeValue);
                } 
                else {
                    cancel();
                }
            }
```

Then in cellEdited, I added a quick fix before the server is sent the response: Have to change the value again to a blank value so that it will resend to the server. Otherwise, it will cause 2 server requests if I dont cancel this one out. 

cellEdited:function(cell){

      if (value=='XREMOVE') {
                    cell.setValue('', true);
                    return false;
                }

     // server response here
}

**Comment by olifolkerd** [comment]
Hey @timothymarois 

This has been resolved on the 4.6 branch, which will be released later today.

Cheers

Oli :)
